tests/integration/test_vectordraw.py

```diff
@@ -0,0 +1,1032 @@
+    """
+
+    def load_scenario(self, path, params=None):
+        scenario = loader.render_template(path, params)
+        self.set_scenario_xml(scenario)
+        self.element = self.go_to_view("student_view")
+        self.exercise = self.element.find_element_by_css_selector(".vectordraw_block")
+
+    def assert_not_present(self, parent, selector, errmsg):
+        try:
+            parent.find_element_by_css_selector(selector)
+        except NoSuchElementException:
+            pass
+        else:
+            self.fail(errmsg)
+
+    def check_hidden_text(self, selector, expected_text):
+        hidden_text = self.browser.execute_script("return $('{}').text();".format(selector))
+        self.assertEquals(hidden_text, expected_text)
+
+    def check_title_and_description(self, expected_title="Vector Drawing", expected_description=None):
+        title = self.exercise.find_element_by_css_selector("h2")
+        self.assertEquals(title.text, expected_title)
+        if expected_description:
+            description = self.exercise.find_element_by_css_selector(".vectordraw-description")
+            self.assertEquals(description.text, expected_description)
+        else:
+            self.assert_not_present(
+                self.exercise,
+                ".vectordraw-description",
+                "Description element present even though no description has been set by user."
+            )
+
+    def check_dimensions(self, board, expected_width="550px", expected_height="400px"):
+        width = board.value_of_css_property("width")
+        height = board.value_of_css_property("height")
+        self.assertEquals(width, expected_width)
+        self.assertEquals(height, expected_height)
+
+    def check_axis(self, board, is_present=False):
+        text_elements = board.find_elements_by_css_selector(".JXGtext")
+        ticks = any("ticks" in text_element.get_attribute("id") for text_element in text_elements)
+        self.assertEquals(ticks, is_present)
+
+    def check_navigation_bar(self, board, is_present=False):
+        if is_present:
+            navigation_bar = board.find_element_by_css_selector("#jxgboard1_navigationbar")
+            self.assertTrue(navigation_bar.is_displayed())
+        else:
+            self.assert_not_present(
+                board,
+                "#jxgboard1_navigationbar",
+                "Navigation bar should be hidden by default."
+            )
+
+    def check_background(self, board, is_present=False):
+        if is_present:
+            background = board.find_element_by_css_selector("image")
+            self.assertTrue(background.is_displayed())
+            src = background.get_attribute("xlink:href")
+            self.assertEquals(src, "https://github.com/open-craft/jsinput-vectordraw/raw/master/Notes_and_Examples/2_boxIncline_multiVector/box_on_incline.png")
+            alt = background.get_attribute("alt")
+            self.assertEquals(alt, "A very informative description")
```

Comment:
@singingwolfboy The description is configurable in Studio via the `background_description` field (cf. [`editable_fields`](https://github.com/open-craft/xblock-vectordraw/blob/initial-implementation/vectordraw/vectordraw.py#L234) and [`drawBackground`](https://github.com/open-craft/xblock-vectordraw/blob/initial-implementation/vectordraw/public/js/vectordraw.js#L55)); this is just a [dummy value](https://github.com/open-craft/xblock-vectordraw/blob/initial-implementation/tests/integration/xml/custom.xml#L15).